src/app/games/yinsh/yinsh.component.ts

```diff
@@ -339,10 +340,10 @@ export class YinshComponent extends HexagonalGameComponent<YinshRules,
     }
 
     private moveToCaptureSelectLast(possibleCaptures: YinshCapture[]): void {
-        this.selectableCoords = [];
+        this.selectableCoords.clear();
```

Comment:
mutabileur !!

Reply:
j'ai transformé une variable mutable d'array  mutable en variable immutable de set mutable, donc immutabileur !

Reply:
J'aimerais mieux des variables mutables de sets immutables !

Reply:
tu préfères un = new MGPSet() plutôt qu'un .clear() ?

Reply:
En vrai faudrait surtout remplacer MGPSet par MutableSet et ImmutableSet, on on utiliserais le premier que temporairement avant d'avoir le deuxième, ils ne seraient pas mutuellement assignable sans passer par .toMutable/.toImmutable. ça ce serait fantastique non ?

Reply:
oui, on a déjà un ticket pour ça ! [W] [easy] MGPSet: Remove "makeImmutable" and split MGPSet into MutableSet and ImmutableSet